engine/crash/wscript

```diff
@@ -9,17 +9,23 @@ blddir = 'build'
 import sys, re, os
 import waf_dynamo, waf_ddf
 from BuildUtility import BuildUtility, BuildUtilityException, create_build_utility
+import waflib.Options
+import TestContext
 
-def init():
-    pass
+test_context = None
 
-def set_options(opt):
-    opt.tool_options('waf_dynamo')
-    opt.tool_options('waf_ddf')
+def init(ctx):
+    global test_context
+    if 'build' in waflib.Options.commands:
+        test_context = TestContext.create_test_context()
```

Comment:
This pattern is now in every top level waf file due to changes in waf. When a command has finished it is removed from the commands list, so we need to look at the list before we build otherwise we don't know if we have to build (and run) tests later in the chain. Hence the "test context" that we create here